game/src/edit/traffic_signals/mod.rs

```diff
@@ -221,7 +221,7 @@ impl State for TrafficSignalEditor {
                     let idx = x.parse::<usize>().unwrap() - 1;
                     return Transition::Push(edits::ChangeDuration::new(
                         ctx,
-                        canonical_signal.stages[idx].phase_type.clone(),
+                        canonical_signal.clone(),
```

Comment:
`ChangeDuration::new` could just take `&ControlTrafficSignal`, it doesn't need a copy to own

Reply:
Yup, I was just going off the original statement, but it's a unnecessary clone